pkg/pluginmanager/manager.go

```diff
@@ -767,13 +761,17 @@ func installOrUpgradePlugin(p *discovery.Discovered, version string, installTest
 		}()
 
 		// Initialize the spinner
-		sow, spinnerErr = component.NewOutputWriterSpinnerWithSpinnerOptions(os.Stderr, component.TableOutputType, installingMsg, true, spinnerOptions)
+		spinner, spinnerErr = component.NewSpinner(component.WithStdout(os.Stderr),
+			component.WithOutputFormat(component.TableOutputType),
```

Comment:
I guess this is something we don't need to set as per your [comment](https://github.com/vmware-tanzu/tanzu-plugin-runtime/pull/157#discussion_r1477468591)